RUNIT TEST PROTOCOL -- Tue Sep 26 08:31:16 2017 
*********************************************** 
Number of test functions: 2 
Number of errors: 0 
Number of failures: 1 

 
1 Test Suite : 
packager_unit_test - 2 test functions, 0 errors, 1 failure
FAILURE in test_exception: Error in RUnit::checkIdentical(result, expected, msg = "Value of digest::sha1() differs!") : 
  FALSE 
 Value of digest::sha1() differs!



Details 
*************************** 
Test Suite: packager_unit_test 
Test function regexp: ^test_+ 
Test file regexp: ^.*\.[rR] 
Involved directory: 
/home/nik/git/cs/fvafrcu/packager/tests/runit 
--------------------------- 
Test file: /home/nik/git/cs/fvafrcu/packager/tests/runit/main.R 
test_exception: FAILURE !! (check number 1)
Error in RUnit::checkIdentical(result, expected, msg = "Value of digest::sha1() differs!") : 
  FALSE 
 Value of digest::sha1() differs!
--------------------------- 
Test file: /home/nik/git/cs/fvafrcu/packager/tests/runit/runit-throw.R 
test_exception: (1 checks) ... OK (0 seconds)
